xblock/fields.py

```diff
@@ -752,7 +766,9 @@ def __init__(self, help=None, default=UNSET, scope=Scope.content, display_name=N
                                       **kwargs)
 
     def from_json(self, value):
-        if isinstance(value, basestring):
+        if isinstance(value, six.binary_type):
+            value = value.decode('latin-1')  # guaranteed successful decode
```

Comment:
latin-1 is guaranteed to decode, because every possible byte value can be represented as a unicode value.  If the value were encoded as something else, it would decode as a different codepoint than intended, but it would not lose any data.  In this instance, that doesn't matter, because for any codepoint outside of ascii, `value.lower()` will not equal `'true'`.  

I was just made aware yesterday of the `errors` argument to `bytes.decode()`, which would make the intention clearer: `value.decode('ascii', errors='replace')`.  I will update this.